### ISSUE TITLE
Unable to remove chat from chat list in landscape

### ISSUE THREAD

**Issue by mocknotes** [urbit/urbit#2504]
**Describe the bug**
Unable to remove chat from chat list in landscape.

**To Reproduce**
1. User 1 creates a chat and user 2 joins
2. Verify user 2 has been added to the chat
3. User 1 deletes the chat before user 2 leaves the chat
4. User 2 goes to chat settings in landscape and selects the `Leave Chat` button
5. Leaving chat hangs and doesn't complete

**Expected behavior**
Pressing the `Leave Chat` button should remove the chat from the chat list.

**Screenshots**
![Screen Shot 2020-03-14 at 1 45 18 AM](https://user-images.githubusercontent.com/822317/76678783-fb5f1100-6597-11ea-8037-379d804d77e1.png)

**Desktop (please complete the following information):**
 - OS: MacOS 10.15.3
 - Browser: chrome
 - Base hash: 0v8.rpvar.1t9h7.4es5r.irauk.niaqo.7do90.bpc1r.nihtd.ufdgt.8ehjl

**Comment by mocknotes** [comment]
This is the console log in the screenshot above

```
Warning: Encountered two children with the same key, `0v8.3641i.2nj0r.05kav.433cv.7h7q4`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
    in div (at chat.js:267)
    in div (at chat.js:233)
    in Jh (at root.js:183)
    in div (at skeleton.js:67)
    in div (at skeleton.js:37)
    in div (at skeleton.js:34)
    in Rh (at root.js:176)
    in t (at root.js:139)
    in div (at root.js:66)
    in t (created by t)
    in t (at root.js:65)
    in L (at src/index.js:15)
```
```
Uncaught (in promise) /~sampel-palnet/home/0/app/chat-view/hoon:<[367 28].[367 73]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[367 5].[368 30]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[359 5].[368 30]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[353 5].[368 30]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[352 5].[368 30]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[223 21].[223 51]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[223 5].[238 7]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[202 3].[248 5]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[201 3].[248 5]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[200 3].[393 5]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[199 3].[393 5]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[89 7].[89 69]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[88 7].[89 69]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[79 5].[94 7]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[78 5].[94 7]>
/~sampel-palnet/home/0/app/chat-view/hoon:<[77 5].[94 7]>
/~sampel-palnet/home/0/lib/dbug/hoon:<[31 25].[31 47]>
/~sampel-palnet/home/0/lib/dbug/hoon:<[31 7].[32 19]>
/~sampel-palnet/home/0/lib/dbug/hoon:<[30 5].[74 7]>
/~sampel-palnet/home/0/lib/dbug/hoon:<[29 5].[74 7]>
/~sampel-palnet/home/0/lib/verb/hoon:<[38 21].[38 43]>
/~sampel-palnet/home/0/lib/verb/hoon:<[38 3].[39 15]>
/~sampel-palnet/home/0/lib/verb/hoon:<[33 3].[39 15]>
/~sampel-palnet/home/0/lib/verb/hoon:<[32 3].[39 15]>
/~sampel-palnet/home/0/lib/verb/hoon:<[31 3].[39 15]>
/~sampel-palnet/home/~2020.3.13..03.40.10..bf43/sys/vane/gall:<[1.191 9].[1.191 37]>
```

**Comment by vvisigoth** [comment]
Thanks, this is a good one. @loganallenc is this already logged?

**Comment by Fang-** [comment]
I think this was fixed as per @ixv's request on #2546. Closing as such, feel free to reopen if the problem persists after we push that out OTA!
